packages/builder/src/helpers/routes.ts

```diff
@@ -0,0 +1,44 @@
+interface BuilderWorkspaceRouteOptions {
+  applicationId: string
+  segments?: Array<string | null | undefined>
+}
+
+const trimSlashes = (value: string) => value.replace(/^\/+|\/+$/g, "")
```

Comment:
NAB: I always like an example of the regex replacement in a comment.

Reply:
Example added